src/raw/lined/phone-missed.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.997 20" width="24" height="24" stroke="currentColor">
```

Comment:
This is a duplicate of `hangup`. I would delete `hangup` as the file name doesn't follow the naming convention of `phone-*`

Reply:
Hangup deleted